### ISSUE TITLE
Stop using the public beta API key

### ISSUE THREAD

**Issue by ghost** [vdemedes/gifi#7]
Aloha,

The public beta API key (still in use in `gifi`) can be _rate-limited_ in some ways, and it's advised not to use it in any production application (see https://github.com/Giphy/GiphyAPI#public-beta-key for more details).

Don't you think it would be nice to add at least one paragraph in `Readme.md` about how to get one _personnal_ API key?  And why not handle the errors in case of reached rate-limit and deport the key in a configuration file.

**Comment by vadimdemedes** [comment]
Yeah, I am aware of this, I didn't want to wait for my application to be accepted. I will definitely replace it with my own API key. As for personal API key, I don't think users want to apply for API keys themselves just to use a silly `npm install` wrapper.
